README.md

```diff
@@ -45,11 +45,24 @@ See the `examples` folder for examples.
 
 ## Development
 
+### Unit tests
+
 In order to test locally, run:
 ```bash
 python setup.py test
 ```
 
+### Documentation
+Run
+
+```bash
+cd docs
+make html
+```
+
+which will make Sphinx update the documentation HTML pages are `_build/html`: just spawn
```

Comment:
nit:
"to update the HTML documentation under `_build/html`. "
"Spawn an HTTP server in that folder to see..."

Reply:
done